### DISCUSSION TITLE
Printed a case to house all the components

### DISCUSSION THREAD

**Discussion by gltovar** [tesla-android/issue-tracker#71] [Ideas]
I printed out a case to house the two Pis and all the components.  I'll write a more in depth post a bit later but I'd figure I'd share this here first :) https://imgur.com/a/5Y8wefJ

**Comment by ochompsky** [comment]
case looks good but two usb-c connectors would have been far less messy than that jumble of power wires

**Reply by gltovar** [reply]
I dont plan on having it plugged into the cigarette lighter for long term, and the dual 12v to 5v 3a converters are the majority of the wire mess, I dont think it would save me much to target usbc.  Also if I could nix the hdmi cord then I would have a slimmer profile ;P
